.github/workflows/upgrade-tests.yml

```diff
@@ -1,9 +1,11 @@
 name: Upgrade Tests
 
 on:
-  schedule:
-    - cron: '0 4 * * *'
-  workflow_dispatch:
+  pull_request:
+  push:
+    paths-ignore:
+    # ignore top-level markdown files (CHANGELOG.md, README.md, etc.)
+    - '*.md'
```

Comment:
We've been trying to only run `push` jobs against release branches. CI systems run `pull_request` jobs against the merge result of the branch being PR'd + the branch we're PR'ing to making running against the branch being PR'd on it's on redundant.

In cases where folks want to see CI run without a PR we suggest using draft PR's